TwoNote/app/src/main/res/values/strings.xml

```diff
@@ -18,4 +18,7 @@
     <string name="selected">selected</string>
     <string name="load_error_note">Error: loaded file is not of type Note</string>
     <string name="load_error_dir">Error: loaded file is not of type DirEntry</string>
+    <string name="welcome">Welcome to Two Note</string>
```

Comment:
we never talked about this, but should our app be named "TwoNote" or "Two Note"?

Reply:
that is a good question, if we're following OneNote then I guess TwoNote? I don't mind either way